### ISSUE TITLE
Butter Project Keeps Loading

### ISSUE THREAD

**Issue by susanto-tm** [butterproject/butter-desktop#688]
I've recently installed butter project using npm and there wasn't any issues following the instructions on the documentation. However after I ran "gulp run", butter opens but just kept loading. I tried waiting for about 30 minutes but it just kept loading. The settings tab work though, but if I clicked anything but movie it just shows a blank black screen.

Steps to Reproduce

1. Git clone and install the source code using npm
2. Run "gulp install" and "gulp build"
3. Running "gulp run" will open butter but will not load

**Expected behavior:** [What you expect to happen]
After running "gulp run" the application should load for a while and show the contents within the app.
**Actual behavior:** [What actually happened]
The butter application just keeps loading and shows a blank screen at other pages other than 'movies'
### Technical details
 Windows 10 Pro

**Comment by lynxaegon** [comment]
Yeah, same here but on MacOS High Sierra 10.13.1. I tracked the error to
`generic.js:63 Couldn't find provider archive`
